cmd/querygen/main.go

```diff
@@ -33,7 +33,11 @@ func main() {
 }
 
 func parseTemplates() error {
-	grpcTemplatePtr, err := template.ParseFiles("cmd/querygen/templates/grpc_template.tmpl")
+	// Create a function to upper vase the version suffix if it exists.
+	funcMap := template.FuncMap{
+		"ToUpper": strings.ToUpper,
+	}
+	grpcTemplatePtr, err := template.New("grpc_template.tmpl").Funcs(funcMap).ParseFiles("cmd/querygen/templates/grpc_template.tmpl")
```

Comment:
Useless knowledge that I now have: template name in `New` must match file name in `ParseFiles`.
https://stackoverflow.com/questions/49043292/error-template-is-an-incomplete-or-empty-template